ts/main.ts

```diff
@@ -78,15 +73,55 @@ function swapView(anime: Anime): void {
     '.detail-rating h2',
   ) as HTMLHeadingElement;
 
+  // Update title
   titleElement.textContent = anime.title;
+
+  // Update synopsis
   synopsisElement.textContent = anime.synopsis;
-  imageElement.innerHTML = `<img src="${anime.images.jpg.image_url}" alt="${anime.title}">`;
+
+  // Update image
+  const img = document.createElement('img');
+  img.src = anime.images.jpg.image_url;
+  img.alt = anime.title;
+
+  // Clear existing children in imageElement and append new image
+  while (imageElement.firstChild) {
+    imageElement.removeChild(imageElement.firstChild);
+  }
+  imageElement.appendChild(img);
+
```

Comment:
It would be more ideal to just change the src of the image than delete and add a new one. If it is showing the old image then you would need to have something that resets the image and text content when switching views back to the all anime view.